js-miniapp-sample/README.md

```diff
@@ -0,0 +1,133 @@
+# miniapp-build-common
+
+This is the boilerplate for the MiniAppBuild Team.
+
+The project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Table of contents
+
+- [miniapp-build-common](#miniapp-build-common)
+  - [Table of contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Installation](#installation)
+  - [Testing](#testing)
+  - [Building](#building)
+  - [Export Dockerized App Image as TAR](#export-dockerized-app-image-as-tar)
+  - [Import Dockerized App Image from TAR](#import-dockerized-app-image-from-tar)
+  - [Flow](#flow)
+  - [Custom Environment Variables](#custom-environment-variables)
+  - [Code Formatting](#code-formatting)
+  - [Naming Convention](#naming-convention)
+    - [Branches](#branches)
+    - [Commits](#commits)
+  - [Branching policy](#branching-policy)
+  - [Others](#others)
+
+## Requirements
+
+- If you're a new member **[please read the MiniAppBuild guidelines](https://confluence.rakuten-it.com/confluence/display/ONEAPP/MAB+-+On-boarding+Guide+for+Engineers)**
```

Comment:
Not sure if there are any security concerns with exposing our internal Confluence URLs. Also, since the repo is moved to a different host, I believe the below requirement about the SSH keys is no longer needed.